administrator/components/com_joomgallery/views/categories/view.html.php

```diff
@@ -82,7 +82,7 @@ protected function addToolbar()
       JToolbarHelper::divider();
     }
 
-    if(($this->_config->get('jg_disableunrequiredchecks') || $canDo->get('core.delete') || count(JoomHelper::getAuthorisedCategories('core.delete'))) && $this->pagination->total)
+    if(($this->_config->get('jg_disableunrequiredchecks') || $canDo->get('core.delete') || $canDo->get('joom.delete.own') || count(JoomHelper::getAuthorisedCategories('core.delete'))) && $this->pagination->total)
```

Comment:
In the function JoomHelper::getAuthorisedCategories($action) you should add
```
      if($action == 'core.delete')
      {
        $action2 = 'core.delete.own';
      }
```

Reply:
Do you mean 'joom.delete.own' for action2 ?

Reply:
> Do you mean 'joom.delete.own' for action2 ?

Yes, that's what I meant. My fault.